decidim-core/app/controllers/decidim/application_controller.rb

```diff
@@ -21,6 +21,9 @@ class ApplicationController < ::DecidimController
     include UserBlockedChecker
     include DisableRedirectionToExternalHost
 
+    include RedesignLayout
```

Comment:
@entantoencuanto could this be removed?

Reply:
The `redesign active: false` can be removed, but the concern has to be included